x-pack/plugin/ilm/src/test/java/org/elasticsearch/xpack/slm/SlmHealthIndicatorServiceTests.java

```diff
@@ -122,14 +136,154 @@ public void testIsGreenWhenNoMetadata() {
                     GREEN,
                     "No Snapshot Lifecycle Management policies configured",
                     null,
-                    new SimpleHealthIndicatorDetails(Map.of("slm_status", RUNNING, "policies", 0)),
+                    new SimpleHealthIndicatorDetails(
+                        Map.of("slm_status", RUNNING, "policies", 0, "unhealthy_policies", 0, "invocations_since_last_success", Map.of())
+                    ),
                     Collections.emptyList(),
                     Collections.emptyList()
                 )
             )
         );
     }
 
+    public void testIsGreenWhenPoliciesHaveFailedForLessThanWarningThreshold() {
+        long execTime = System.currentTimeMillis();
+        long window = TimeUnit.HOURS.toMillis(24) - 5000L; // Just under 24 hours.
+        var clusterState = createClusterStateWith(
+            new SnapshotLifecycleMetadata(
+                createSlmPolicyWithInvocations(
+                    snapshotInvocation(execTime, execTime + 1000L),
+                    snapshotInvocation(execTime + window, execTime + window + 1000L),
+                    randomLongBetween(0, 4)
+                ),
+                RUNNING,
+                null
+            )
+        );
+        var service = createSlmHealthIndicatorService(clusterState);
+
+        assertThat(
+            service.calculate(true),
+            equalTo(
+                new HealthIndicatorResult(
+                    NAME,
+                    SNAPSHOT,
+                    GREEN,
+                    "Snapshot Lifecycle Management is running",
+                    null,
+                    new SimpleHealthIndicatorDetails(
+                        Map.of("slm_status", RUNNING, "policies", 1, "unhealthy_policies", 0, "invocations_since_last_success", Map.of())
```

Comment:
This strikes me as kind of odd -- the result is correctly GREEN because there have been 4 or fewer failures since the last success. And "unhealthy_policies" is correctly 0 for the same reason. But "invocations_since_last_success" is missing all the invocations since the last success. Should we be putting that data there even when it's GREEN if they ask for explain?